service/awsconfig/v5/resource/loadbalancer/delete.go

```diff
@@ -2,13 +2,32 @@ package loadbalancer
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
 )
 
-// TODO
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+	deleteInput, err := toLoadBalancerState(deleteChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if deleteInput != nil && len(deleteInput.LoadBalancerNames) > 0 {
+		for _, lbName := range deleteInput.LoadBalancerNames {
+			if _, err := r.clients.ELB.DeleteLoadBalancer(&elb.DeleteLoadBalancerInput{
+				LoadBalancerName: aws.String(lbName),
+			}); err != nil {
+				return microerror.Mask(err)
+			}
```

Comment:
For readability I would find it nice to separate function calls and conditionals.

Reply:
Good call. See https://github.com/giantswarm/aws-operator/pull/762